include/swift/AST/Decl.h

```diff
@@ -4325,15 +4325,28 @@ class AbstractStorageDecl : public ValueDecl {
 
 /// VarDecl - 'var' and 'let' declarations.
 class VarDecl : public AbstractStorageDecl {
+public:
+  enum class Specifier : uint8_t {
+    // For Var Decls
+    
+    Let  = 0,
+    Var  = 1 << 0,
+    
+    // For Param Decls
+    
+    None  = Let,
+    InOut = 1 << 1,
```

Comment:
Nitpick: Since these aren't meant to be ORed together, regular numbers might be clearer than shifts.